src/main/java/com/everis/boundary/CarResources.java

```diff
@@ -82,7 +85,9 @@ public Response updateCar(final @PathParam("id") String id, final CarDto cardto)
 		
 		LOGGER.info("Validating Car's info: " + cardto);
 		try {
+			
 			CarDto updatedCar=carService.updateCar(id, Car.MapToCar(cardto));
+			carMessagePublisher.carEditionJms(Car.MapToCar(cardto));
```

Comment:
Here you're updating the car two times fist with carService.updateCar and second with the message listener. Please double check this.